File: benchmark/corpus/03_div_zero/c_per_item_in_main.js
// The division is in main on an input-derived expression: (n - 1) is 0 when n is 1.
const line = (await new Response(Deno.stdin.readable).text()).trim();
const budget = 100;
const n = Number(line);
if (Number.isNaN(n)) console.log("not a number: " + line);
else {
  const each = Math.floor(budget / (n - 1)); // DANGER
  console.log("each of the others gets " + each);
}

File: benchmark/corpus/03_div_zero/c_per_item_in_main.py
# The division is in main on an input-derived expression: (n - 1) is 0 when n is 1.
import sys

line = sys.stdin.readline().strip()
budget = 100
try:
    n = int(line)
except ValueError:
    print("not a number: " + line)
else:
    each = budget // (n - 1)  # DANGER
    print("each of the others gets %d" % each)
